test/test_shell_script.py

```diff
@@ -30,7 +30,8 @@
     "--mhc-predictor", "random",
     "--mhc-alleles", "H2-Kb,H2-Db",
     "--padding-around-mutation", "5",
-    "--include-mismatches-after-variant"
+    # TODO: figure out what happened to this argument in Isovar
```

Comment:
?

Reply:
Found the new name, it's   --count-mismatches-after-variant